easybuild/easyconfigs/p/Pindel/Pindel-0.2.5b8-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+
+easyblock = 'CmdCp'
+
+name = 'Pindel'
+version = '0.2.5b8'
+
+homepage = 'http://gmt.genome.wustl.edu/packages/pindel/'
+description = """ Pindel can detect breakpoints of large deletions, medium sized 
+ insertions, inversions, tandem duplications and other structural variants at single-based 
+ resolution from next-gen sequence data. It uses a pattern growth approach to identify the 
+ breakpoints of these variants from paired-end short reads. """
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+toolchainopts = {'openmp': True}
+
+# required download of specific PR to v0.2.5b8 to fix compilation error
+# see https://github.com/genome/pindel/pull/64
+source_urls = ['https://github.com/genome/pindel/archive/']
+sources = ['70c1bb4a75503da39e206e02178fe3d8a0afdf81.zip']
```

Comment:
@jdemeul  You should rename the downloaded zipball, at the very least, cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources .

However, in this case I think it makes more sense to download the PR separately as a proper patch, something like this (untested, I'll leave that up to you ;-)):

```python
source_urls = [
    'https://github.com/genome/pindel/archive/',
    'https://patch-diff.githubusercontent.com/raw/genome/pindel/pull/',
]
sources = ['v%(version)s.tar.gz']
# apply changes made in https://github.com/genome/pindel/pull/64 to fix compilation error
patches = ['64.patch']
checksums = [ ... ]
```

(please use `eb --inject-checksums` to also receive a checksum for the patch file)